package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --proxy-config proxy.conf.json",
```

Comment:
--proxy-config proxy.conf.json 
to już znajduje się w pliku angular.json

Reply:
tak, to draft który wystawiłem tylko na potrzeby rozwiązania problemów z routingiem i konfliktów jest tu troche niepotrzebnych tematów